java/quarkus/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>benchmarker.the</groupId>
+  <artifactId>quarkus</artifactId>
+  <version>1.0.0</version>
+  <properties>
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <maven.compiler.parameters>true</maven.compiler.parameters>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+    <quarkus-plugin.version>1.11.0.Final</quarkus-plugin.version>
```

Comment:
is there any parity between `quarkus.platform.version` and `quarkus-plugin.version` ?

Reply:
I would think so, yeah. Maybe @geoand can confirm.

Reply:
Yes, those need to match

Reply:
then you could use only one version (`quarkus.platform.version`) instead of two vars in `pom.xml` I suppose @whiplash

Reply:
Yeah, that's pretty common for users to do

Reply:
Done